tsconfig.build.json

```diff
@@ -1,8 +1,8 @@
 {
 	"compilerOptions": {
 		"module": "NodeNext",
-		"target": "es2021",
-		"lib": ["es2021"],
+		"target": "ESNext",
```

Comment:
I updated this so that I could use the non-destructive `Array#toSorted` instead of the destructive `Array#sort`.

Reply:
IS ES2022 sufficient? I think ESNext currently is ES2022 but we might want to lock in 2022 instead of having this changed underneith us without warning some day.

https://github.com/forcedotcom/code-analyzer-core/blob/dev/tsconfig.base.json is an example of what I did

Reply:
Unfortunately, there's no `ES2023` option available, so we're stuck with `ESNext` for now. Alternatively, I could refactor so we copy the list and sort the copy instead of calling `Array.toSorted`.

Reply:
Well if you don't need a copy... then why not just use the sort method instead? The toSorted returns a new copy of the array where the sort method (which exists in ES2022) just sorts the array in place (which is more efficient).